boards/nxp/mimxrt1050_evk/revision.cmake

```
@@ -0,0 +1,7 @@
+if (NOT DEFINED BOARD_REVISION)
+	set(BOARD_REVISION "hyperflash")
+else ()
+	if (NOT (BOARD_REVISION STREQUAL "hyperflash") AND NOT (BOARD_REVISION STREQUAL "qspi"))
+		message(FATAL_ERROR "Invalid board revision")
```

as a courtesy to the user, who might simply have made a typo, maybe print what was invalid, along with valid revisions.
```suggestion
    message(FATAL_ERROR "Invalid board revision, ${BOARD_REVISION}, valid revisions are: hyperflash, qspi")
```

btw. CMake files uses spaces for indentaiton, not tabs.